playbooks/roles/caasp4/templates/skuba-terraform-openstack.tfvars.j2

```diff
@@ -29,7 +29,7 @@ packages = [
   "kernel-default",
   "-kernel-default-base",
   "ca-certificates-suse",
-  "patterns-caasp-Node",
+  "patterns-caasp-Node-1.14-1.15",
```

Comment:
Did that package rename happen by intention? Maybe the package misses a Priovides?

Reply:
Yes, this was done on purpouse between beta 4 and 5: https://susedoc.github.io/caasp-release-notes/beta/release-notes/single-html/#_updating_from_beta_4_to_beta_5

Seems like its due to architectural changes and the package names are different because you need a manual step to upgrade. In our case we should go with 1.15 directly